lib/firmware/ethereum.c

```diff
@@ -724,10 +770,13 @@ void ethereum_signing_init(EthereumSignTx *msg, const HDNode *node,
   }
 
   memset(confirm_body_message, 0, sizeof(confirm_body_message));
-  layoutEthereumFee(msg->value.bytes, msg->value.size, msg->gas_price.bytes,
-                    msg->gas_price.size, msg->gas_limit.bytes,
-                    msg->gas_limit.size, token != NULL, confirm_body_message,
-                    sizeof(confirm_body_message));
+  layoutEthereumFee(
+      msg->value.bytes, msg->value.size, msg->gas_price.bytes,
+      msg->gas_price.size, msg->gas_limit.bytes, msg->gas_limit.size,
+      msg->max_fee_per_gas.bytes, msg->max_fee_per_gas.size,
+      msg->max_priority_fee_per_gas.bytes, msg->max_priority_fee_per_gas.size,
+      msg->has_max_fee_per_gas ? true : false, token != NULL,
+      confirm_body_message, sizeof(confirm_body_message));
```

Comment:
Wow, big param list. Is it better just to pass msg rather than a list of everything in msg here?

Reply:
I think that's a better idea. I'll make the change.